lib/supavisor/client_handler.ex

```diff
@@ -101,12 +99,12 @@ defmodule Supavisor.ClientHandler do
     end
   end
 
-  def handle_event(:internal, :subscribe, _, %{tenant: tenant} = data) do
-    Logger.info("Subscribe to tenant #{tenant}")
+  def handle_event(:internal, :subscribe, _, %{tenant: tenant, user_alias: db_alias} = data) do
+    Logger.info("Subscribe to tenant #{inspect({db_alias, tenant})}")
```

Comment:
nitpick, in many other places we'd log `{tenant, db_alias}` and so being consistent here might help debugging later, i.e. searching for the same thing:

```suggestion
    Logger.info("Subscribe to tenant #{inspect({tenant, db_alias})}")
```

Reply:
Yep, right, thanks!